### ISSUE TITLE
Choose fonts

### ISSUE THREAD

**Issue by seanmturley** [seanmturley/natureddit#150]
# Description

Choose and implement fonts.

This could be an iterative process, as I'm not sure I'll get it right the first time.

**Comment by seanmturley** [comment]
Added a first pass at fonts:
- Work Sans for everything else (this looks nice for the tiny detail text as well as the headings)
- Open Sans for body text (more readable than Work Sans for paragraphs)

Some adjustments in font size and line height were also made to improve visual hierarchy and legibility.

This isn't exactly the direct I expected to go in - very minimal and modern, i.e. no "nature" influence. However, I found that the pages are so information dense that more decorative font styles looked overwhelming or just plain messy.

This may be reviewed as I work on other aesthetic improvements, as I'm not yet completely convinced. None the less, I do think the clean, modern direction is a good one.
